File: nodoDoble.js
class nodoDoble
{

    constructor (dato)
    {
        this.dato = dato;
        this.Ld = null;
        this.Li = null;
    }

    set asignaDato (dato)
    {
        this.dato = dato;
    }
        
    get retornaDato()
    {
        return this.dato;
    }

    set asignaLd (Ld)
    {
        this.Ld = Ld;
    }

    set asignaLi (Li)
    {
        this.Li = Li;
    }

    get retornaLd()
    {
        return this.Ld;
    }

    get retornaLi()
    {
        return this.Li;
    }

}

File: LDL/src/LDL/nodoDoble.java
package LDL;

public class nodoDoble
{
    private int dato;
    private nodoDoble Li;
    private nodoDoble Ld;
    
    public nodoDoble (int d)           { dato = d; Ld = null; Li = null; }
    public void asignaDato (int d)     { dato = d;    }
    public int  retornaDato ()         { return dato; }
    public void asignaLd (nodoDoble x) { Ld = x;      }
    public void asignaLi (nodoDoble x) { Li = x;      }
    public nodoDoble retornaLd ()      { return Ld;   }
    public nodoDoble retornaLi ()      { return Li;   }
}
